src/main/scala/swiss/dasch/domain/StorageService.scala

```diff
@@ -22,10 +23,10 @@ import java.time.format.DateTimeFormatter
 import java.time.{ZoneId, ZoneOffset}
 
 trait StorageService {
-  def getProjectDirectory(projectShortcode: ProjectShortcode): UIO[ProjectFolder]
-  def getAssetDirectory(asset: AssetRef): UIO[Path]
-  def getAssetDirectory(): UIO[Path]
-  def getTempDirectory(): UIO[Path]
+  def getProjectFolder(projectShortcode: ProjectShortcode): UIO[ProjectFolder]
+  def getAssetFolder(asset: AssetRef): UIO[AssetFolder]
+  def getAssetsBaseFolder(): UIO[AssetsBaseFolder]
+  def getTempFolder(): UIO[TempFolder]
```

Comment:
What is the motivation behind that rename? I can still see some "dir" / "directory" props floating around.

Reply:
Aligning the method names with the returned types.